gym_snake/envs/snakeRewardFuncs.py

```diff
@@ -20,4 +20,14 @@ def basic_reward_func(reward_dict) -> float:
     elif reward_dict["did_collide_wall"] or reward_dict["did_collide_body"]:
         return -1
     else: 
-        return 0
\ No newline at end of file
+        return 0
+
+def reward_closer_to_fruit(reward_dict) -> float:
+    if reward_dict["did_consume_fruit"]:
```

Comment:
Can we add a comment similar to the `basic_reward_func` one above detailing the intended use of this reward function and arguments it expects? It may just be useful as we add a bunch more, but also totally stylistic.